src/crypto/tls/tls_test.go

```diff
@@ -1071,7 +1071,7 @@ func TestConnectionState(t *testing.T) {
 			name = "TLSv13"
```

Comment:
Instead of modifying an existing test, I think we should create a new file for validating events. E.g., `tls_cf_test.go`? I'm pretty familiar with how to set up a basic handshake test, if you'd like i can push a test case here.

Reply:
I can whip something up. Stand by.